### PR TITLE
Update continuous-integration.md

### PR DESCRIPTION
**Pull request by MarieJM**

### CODE REVIEW

File: docs/best-practices/continuous-integration.md [hunk 1]
```diff
@@ -447,7 +447,7 @@ The extension can be configured to install the latest version of _fastlane_ or a
 ## More Information
 For more documentation, see the [Apple App Store](https://marketplace.visualstudio.com/items?itemName=ms-vsclient.app-store) in the Visual Studio Team Server marketplace.
 
-# Nevercode Integration for iOS
+# Nevercode Integration for iOS & Android
```
Comment (janpio) [inline docs/best-practices/continuous-integration.md]: Now that both platforms are supported, could we maybe move the "supported platforms" bit to the first paragraph?

File: docs/best-practices/continuous-integration.md [hunk 2]
```diff
@@ -483,7 +483,7 @@ In your app settings on Nevercode, navigate to the **Environment** tab to manage
 
 ## Run tests
 
-Testing in Nevercode with _fastlane_ is 100% automatic. By default, [`scan`](https://docs.fastlane.tools/actions/scan/) is used to invoke the test run, but you can easily swap it out for any other `lane` that you have configured for your test runs. Test results are automatically collected and you don't need to do anything to convert the results from one format to another or place them in a special location on the build machine.
+Testing in Nevercode with **_fastlane_ for iOS** is 100% automatic. By default, [`scan`](https://docs.fastlane.tools/actions/scan/) is used to invoke the test run, but you can easily swap it out for any other `lane` that you have configured for your test runs. Test results are automatically collected and you don't need to do anything to convert the results from one format to another or place them in a special location on the build machine.
```
Comment (janpio) [inline docs/best-practices/continuous-integration.md]: Could we make "iOS" and "Android" subheadlines (level `###`) here for better structuring? The text can stay the same, but the bolding can be removed then.

File: docs/best-practices/continuous-integration.md [hunk 3]
```diff
@@ -492,6 +492,16 @@ In case you wish to run tests as part of the main build step, we suggest that yo
 trainer(output_directory: ENV["NEVERCODE_XUNIT_RESULTS_DIR"])
 ```
 
+Nevercode detects your **_fastlane_ tests for Android** automatically. However, tests are disabled by default and have to be enabled manually.
+
+1. In your app settings, navigate to the **Test** tab.
```
Comment (janpio) [inline docs/best-practices/continuous-integration.md]: app settings on nevercode?

File: docs/best-practices/continuous-integration.md [hunk 4]
```diff
@@ -492,6 +492,16 @@ In case you wish to run tests as part of the main build step, we suggest that yo
...
+1. Click on **_fastlane_ Gradle** to expand it.
+1. Select **Enabled** and type the **lane name** for tests as specified in your `Fastfile`.
+1. If you would like to run the tests on Android emulator, make sure to select **Require Android emulator**.
+1. Click **Save** to finish the setup. Your _fastlane_ tests will be enabled and run each time you build your app.
+
+You can also use **AWS Device Farm** for testing.
```
Comment (janpio) [inline docs/best-practices/continuous-integration.md]: Only on Android or for both? If for both, this should probably be moved to an introductory paragraph after "Run tests" that explains how tests are normally run and that AWS is an alternative.

File: docs/best-practices/continuous-integration.md (hunk 1 above)
Comment (MarieJM) [reply docs/best-practices/continuous-integration.md]: Yeah sure,  I will do that:)

File: docs/best-practices/continuous-integration.md (hunk 2 above)
Comment (MarieJM) [reply docs/best-practices/continuous-integration.md]: Good point!

File: docs/best-practices/continuous-integration.md (hunk 3 above)
Comment (MarieJM) [reply docs/best-practices/continuous-integration.md]: Yes. Improved.

**Comment by MarieJM** [comment]
Hei @janpio Thanks for taking your time! (Y) I made all the improvements.

**Comment by janpio** [comment]
Don't worry about the @googlebot message, I will take care of the merging.

I changed a few tiny bits around, please confirm you are ok with that and I can merge.

**Comment by MarieJM** [comment]
Yes, everything is ok from my side. Thank you!!!

**Pull request merged**